src/Command/ConsumerCommand.php

```diff
@@ -51,13 +54,14 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $provider = $broker->getProvider($gate);
         $producer = $broker->getProducer($gate);
 
-        $processor = new CommandProcessor($logger, new ProcessBuilder,
-                                          $provider, $producer,
-                                          $container->getRootDir() . '/app/console',
-                                          $kernel->getEnvironment(),
-                                          true === $input->getOption('disable-verbosity-propagation')
-                                            ? OutputInterface::VERBOSITY_QUIET
-                                            : $output->getVerbosity()
+        $processor = new CommandProcessor(
+            $logger,
+            new ProcessBuilder,
+            $provider,
+            $producer,
+            sprintf('%s %s', $container->getRootDir(), $consolePath),
```

Comment:
With this, it's basically assuming that the `bin/console` (or whatever) is launched from the root dir. e.g you can't have a personnalized structure, and launch `/var/bin/sf-console consume...`